Pipfile

```diff
@@ -0,0 +1,19 @@
+[[source]]
```

Comment:
we can also explicitly point out which python versions are supported here

Reply:
You mean `[requires]` section?

If so, I removed from below description.
If no and you have example, it would be helpful.

https://docs.pipenv.org/en/latest/basics/#general-recommendations-version-control
>Specify your target Python version in your Pipfile’s [requires] section. Ideally, you should only have one target Python version, as this is a deployment tool.

Reply:
yeah, sad but true: https://github.com/pypa/pipenv/issues/1050